reactos/dll/win32/browseui/basebarsite.cpp

```diff
@@ -334,13 +431,53 @@ HRESULT STDMETHODCALLTYPE CBaseBarSite::SetDeskBarSite(IUnknown *punkSite)
         hResult = oleWindow->GetWindow(&ownerWindow);
         if (FAILED_UNEXPECTEDLY(hResult))
             return hResult;
-        m_hWnd = CreateWindow(REBARCLASSNAMEW, NULL, WS_VISIBLE | WS_CHILDWINDOW | WS_CLIPSIBLINGS |
-                    WS_CLIPCHILDREN |
-                    RBS_VARHEIGHT | RBS_REGISTERDROP | RBS_AUTOSIZE | RBS_VERTICALGRIPPER | RBS_DBLCLKTOGGLE |
-                    CCS_LEFT | CCS_NODIVIDER | CCS_NOPARENTALIGN | CCS_NORESIZE, 0, 0, 0, 0, ownerWindow, NULL,
+
+        DWORD dwStyle =  WS_VISIBLE | WS_CHILD | WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_BORDER |
+             RBS_VARHEIGHT | RBS_REGISTERDROP | RBS_AUTOSIZE | RBS_VERTICALGRIPPER | RBS_DBLCLKTOGGLE |
+             CCS_NODIVIDER | CCS_NOPARENTALIGN | CCS_NORESIZE;
+        if (fVertical)
+            dwStyle |= CCS_VERT;
+
+        /* Create site window */
+        HWND tmp = CreateWindowW(REBARCLASSNAMEW, NULL, dwStyle, 0, 0, 0, 0, ownerWindow, NULL,
                     _AtlBaseModule.GetModuleInstance(), NULL);
+
+        /* Give window management to ATL */
+        SubclassWindow(tmp);
+
         SendMessage(RB_SETTEXTCOLOR, 0, CLR_DEFAULT);
         SendMessage(RB_SETBKCOLOR, 0, CLR_DEFAULT);
+
+        /* Create close toolbar and imagelist */
+        toolbarWnd = CreateWindowW(TOOLBARCLASSNAMEW, NULL,
+            WS_VISIBLE | WS_CHILD | WS_CLIPSIBLINGS | WS_CLIPCHILDREN |
+            TBSTYLE_FLAT | TBSTYLE_TRANSPARENT | TBSTYLE_TOOLTIPS | 
+            CCS_NOMOVEY | CCS_NORESIZE | CCS_NOPARENTALIGN | CCS_NODIVIDER
+            , 0, 0, 0, 0, m_hWnd, NULL, _AtlBaseModule.GetModuleInstance(), NULL);
+
+        toolImageList = ImageList_Create(13, 11, ILC_COLOR24 | ILC_MASK, 3, 0);
+
+        hBmp = (HBITMAP)LoadImage(_AtlBaseModule.GetModuleInstance(),
+            MAKEINTRESOURCE(IDB_BANDBUTTONS), IMAGE_BITMAP, 0, 0,
+            LR_LOADTRANSPARENT);
```

Comment:
Is this image supposed to come from browseui?
Or should it be loaded from the theme?